specifyweb/frontend/js_src/lib/components/Forms/Save.tsx

```diff
@@ -232,17 +232,18 @@ export function SaveButton<SCHEMA extends AnySchema = AnySchema>({
           isSaveDisabled &&
           showCarry &&
           showBulkCarry ? (
-            <Label.Inline>
-              <Input.Generic
-                type="number"
-                value={carryForwardAmount}
-                onValueChange={(value): void =>
-                  carryForwardAmount === undefined
-                    ? setCarryForwardAmount(1)
-                    : setCarryForwardAmount(Number.parseInt(value))
-                }
-              />
-            </Label.Inline>
+            <Input.Integer
+              aria-label={formsText.bulkCarryForwardCount()}
+              className="!w-fit"
+              min={1}
+              placeholder="1"
+              value={carryForwardAmount}
+              onValueChange={(value): void =>
+                value === undefined
+                  ? setCarryForwardAmount(1)
+                  : setCarryForwardAmount(value)
+              }
```

Comment:
is value ever undefined? not according to it's typescript type it seems

or did you mean to handle the case when the user erased the value from the field? we don't have that case well in specify now but probably should - `value` would be NaN in such cases, not undefined

Reply:
yes I wanted to handle to case where we have the user erasing the field (like the issue we had in bulk resolve bulk loaned) will change to NaN